addons/medical_feedback/functions/fnc_handleGUIIndication.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+/*
+ * Author: 10Dozen
+ * Handles indication of the fractures, applied tourniquets and splints over Stance indicator.
+ * Draws an icon if there is at least 1 fracture/splint/tourniquet applied.
+ *
+ * Arguments:
+ * 0: Drop indicator <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [false] call ace_medical_feedback_fnc_handleGUIIndication
+ *
+ * Public: No
+ */
+params [["_dropIndicator", false]];
+private _indicatorSlots = [
+    uiNamespace getVariable [QGVAR(stateIndicator1), controlNull],
+    uiNamespace getVariable [QGVAR(stateIndicator2), controlNull],
+    uiNamespace getVariable [QGVAR(stateIndicator3), controlNull]
+];
+
+// --- Removes any indication and exit
+if (_dropIndicator) exitWith {
+    {
+        _x ctrlSetText "";
+    } forEach _indicatorSlots;
+};
+
+// --- Tourniquets
+private _hasTourniquets = {_x > 0} count GET_TOURNIQUETS(ACE_player) > 0;
+private _tourniquetIcon = ["",ICON_TOURNIQUET_PATH] select _hasTourniquets;
+
+// --- Fractures and Splints
+private _fractureSettings = EGVAR(medical,fractures);
+private _fractureIcon = "";
+private _splintIcon = "";
+
+if (_fractureSettings > 0) then {
+    // --- Fractures enabled: check for fracture indication
+    private _hasFractures = {_x > 0} count GET_FRACTURES(ACE_player) > 0;
+    _fractureIcon = ["",ICON_FRACTURE_PATH] select _hasFractures;
+
+    if (_fractureSettings > 1) then {
+        // --- Fractures can not be fully healed: check for splint indication
+        private _hasSplints = {_x isEqualTo -1} count GET_FRACTURES(ACE_player) > 0;
```

Comment:
```suggestion
        private _hasSplints = GET_FRACTURES(ACE_player) findIf {_x isEqualTo -1} != -1;
```

Reply:
It's a bit unclear what syntax is preferred:
A) `X findIf {Y} != -1`
B) `X findIf {Y} > -1`

I see both A and B used across the repo.
Why not 'isNotEqualTo' instead of `!=`? Is it just for specific data types? (found nothing here https://ace3mod.com/wiki/development/coding-guidelines.html)

Reply:
After a quick regex search, it seems that `>` is used more frequently than `!=` in the ACE repo. However, there's no guideline on that.

As for `isNotEqualTo`, it has no benefits over `!=` for number type and it's longer to read.

Reply:
Thanks for explanations.